recipe/meta.yaml

```
@@ -26,7 +26,7 @@ requirements:
   run:
     - python >=3.9
     - numpy >=1.19.2
-    - imagecodecs >=2023.1.23
+    - imagecodecs >=2023.7.10
```

```suggestion
    - imagecodecs >=2023.8.12
```
According to https://github.com/cgohlke/tifffile/commit/19dfae260223c8d190fc483c7e8e6a6fcb2e0151? However, this version is not available yet: https://github.com/conda-forge/imagecodecs-feedstock/pull/86.

---

imagecodecs 2023.8.12 is available now. so lets go with `imagecodecs >=2023.8.12`